src/components/Alert.tsx

```diff
@@ -119,6 +115,7 @@ export type AlertRef = {
 export type AlertAction = {
   label: string
   onPress?: (AlertRef) => void
+  onPressDismiss?: (AlertRef) => void
```

Comment:
**Question:**
The alert was causing a lot of confusion until I released it's set to close after running the onPress method,  I looked at changing`onPress` to `: (AlertRef) => boolean` and only closing the alert if it returns true however alerts are already used in quite a few places so a seperate method seemed to work better.
Otherwise I could change this to a boolean prop something like `closeAlertAfterPress`.